src/main/java/com/google/javascript/gents/RemoveGoogScopePass.java

```diff
@@ -0,0 +1,69 @@
+package com.google.javascript.gents;
+
+import com.google.javascript.jscomp.AbstractCompiler;
+import com.google.javascript.jscomp.CompilerPass;
+import com.google.javascript.jscomp.NodeTraversal;
+import com.google.javascript.rhino.Node;
+import com.google.javascript.rhino.Token;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nullable;
+
+/**
+ * Compiler pass removing the {@code goog.scope} wrapper, so that the module and class
+ * transformations can take effect.
```

Comment:
Add a comment that this translation is legal, because we're turning all files into modules so `var`s in the `goog.scope` body remain local (not global)?